deeplabcut/gui/tabs/create_project.py

```diff
@@ -48,16 +51,24 @@ def lay_out_user_frame(self):
 
         proj_label = QtWidgets.QLabel("Project:", user_frame)
         self.proj_line = QtWidgets.QLineEdit(self.proj_default, user_frame)
+        self.proj_line.setPlaceholderText("my project's name")
```

Comment:
I'd find it more intuitive that I needed to fill out the text if the placeholder had a different font color (e.g. a light grey, or simply give it reduced opacity)

Reply:
It should; can you try to update PySide6? If that solves it we can probably remove the pin.

Reply:
bump @n-poulsen

Reply:
upgrading PySide6 from 6.3.1 to 6.5.1.1 fixed the issue 👍

Reply:
Though now when using PySide6 6.5.1.1 I can no longer label frames:

```
Traceback (most recent call last):
  File ".../deeplabcut/gui/tabs/label_frames.py", line 59, in label_frames
    _ = launch_napari(folder)
  File ".../deeplabcut/gui/widgets.py", line 38, in launch_napari
    viewer = napari.Viewer()
  File ".../lib/python3.9/site-packages/napari/viewer.py", line 67, in __init__
    self._window = Window(self, show=show)
  File ".../lib/python3.9/site-packages/napari/_qt/qt_main_window.py", line 463, in __init__
    self._qt_window = _QtMainWindow(viewer)
  File ".../lib/python3.9/site-packages/napari/_qt/qt_main_window.py", line 88, in __init__
    self._qt_viewer = QtViewer(viewer, show_welcome_screen=True)
  File ".../lib/python3.9/site-packages/napari/_qt/qt_viewer.py", line 217, in __init__
    self._create_canvas()
  File ".../lib/python3.9/site-packages/napari/_qt/qt_viewer.py", line 401, in _create_canvas
    self.canvas = VispyCanvas(
  File ".../lib/python3.9/site-packages/napari/_vispy/canvas.py", line 33, in __init__
    super().__init__(*args, **kwargs)
...
  File ".../lib/python3.9/site-packages/vispy/app/backends/_qt.py", line 283, in _set_config
    glformat.setSwapBehavior(glformat.DoubleBuffer if c['double_buffer']
AttributeError: 'PySide6.QtGui.QSurfaceFormat' object has no attribute 'DoubleBuffer'
```

Reply:
That was my fear... napari is about to release 0.4.18, I guess they will have addressed most small bugs related to pyside/pyqt6.

Reply:
any updates on napari release?

Reply:
excellent! can me marked as non-draft now?

Reply:
Will do after next napari-deeplabcut release